src/main/scala/devices/debug/Debug.scala

```diff
@@ -1579,13 +1643,14 @@ class TLDebugModuleInner(device: Device, getNComponents: () => Int, beatBytes: I
     val commandWrIsAccessRegister = (COMMANDWrData.cmdtype === DebugAbstractCommandType.AccessRegister.id.U)
     val commandRegIsAccessRegister = (COMMANDReg.cmdtype === DebugAbstractCommandType.AccessRegister.id.U)
 
-    val commandWrIsUnsupported = COMMANDWrEn && !commandWrIsAccessRegister;
+    val commandWrIsUnsupported = COMMANDWrEn && !commandWrIsAccessRegister
```

Comment:
a semi-colon?! that must be a @mwachs5 2017 original 😂

Reply:
When I started at SiFive, it was hard to get out of that habit.  Now, when I write Verilog, I almost always get errors about missing semicolons!